client/my-sites/plans/main.jsx

```diff
@@ -272,10 +270,10 @@ class Plans extends Component {
 				redirectTo={ this.props.redirectTo }
 				withDiscount={ this.props.withDiscount }
 				discountEndDate={ this.props.discountEndDate }
-				site={ selectedSite }
+				siteId={ selectedSite?.ID }
 				plansWithScroll={ false }
 				hidePlansFeatureComparison={ this.props.isDomainAndPlanPackageFlow }
-				planTypes={ planTypes }
+				intent={ isJetpackAppPlans ? 'plans-jetpack-app' : null }
```

Comment:
`Non-blocking / Nit`:

What made us move to the inline ternary? I find it easier to read as it was structured originally with planTypes

```
const intent = this.props.jetpackAppPlans ? 'plans-jetpack-app' : null ;

<PlansFeaturesMain
  {...rest}
  intent={intent}
/>
```

Reply:
Probably something to clean up, as the comment above also remained in the code.

In general, I don't mind _not_ creating additional variables when there's a single use for them. In this case, I see little difference between

`intent={ isJetpackAppPlans ? 'plans-jetpack-app' : null  }`

and 

```
const intent = isJetpackAppPlans ? 'plans-jetpack-app' : null;
...
intent={ intent }
```

or things like 

```
const hideFreePlan = this.props.isDomainAndPlanPackageFlow;
...
hideFreePlan={ hideFreePlan }
```

I totally get it this is fine to do and helps some folks with reading "React code" I guess. However, I've seen things get off hand when everything is tried to be reassigned before assigned to props. verbose variables, thinking ahead (that something may become more "complex" to compute down the line), etc. It can feel noisy.

Reply:
doing it like you mentioned 👍